integration_tests/pages/index.ts

```diff
@@ -12,7 +12,6 @@ export default class IndexPage extends Page {
   }
 
   clickCreateALicence = (): CaseloadPage => {
-    cy.task('stubGetStaffDetails')
     cy.task('stubGetManagedOffenders')
```

Comment:
This is no longer retrieved in the caseload page - already present in the session and passed into the service now.